net-test/bin/start.sh

```diff
@@ -0,0 +1,367 @@
+   local RC=$?
+   local BITCOIN_NEON_CONTROLLER_PID=$!
+
+   if [ $RC -ne 0 ]; then 
+      logln "FAILED"
+      return 1
+   fi
+
+   logln "PID $BITCOIN_NEON_CONTROLLER_PID"
+
+   echo "$BITCOIN_NEON_CONTROLLER_PID"
+   return 0
+}
+
+start_faucet() {
+   logln "Setting up bitcoin faucet..."
+   test -f "$BITCOIN_CONTROLLER_CONF" || exit_error "No such file or directory: $BITCOIN_CONTROLLER_CONF"
+   test -f "$BITCOIN_CONF" || exit_error "No such file or directory: $BITCOIN_CONF"
+
+   local PRIVKEY="$(cat "$BITCOIN_CONTROLLER_CONF" | grep "# WIF" | cut -d ' ' -f 3)"
+
+   log "  Importing faucet private key..."
+   bitcoin-cli -conf="$BITCOIN_CONF" importprivkey "$PRIVKEY" >"$FAUCET_LOGFILE" 2>&1
+   local RC=$?
+   if [ $RC -ne 0 ]; then
+      logln "FAILED!"
+      return 1
+   fi
+   logln "ok"
+
+   local MINER_ADDR="$(cat "$BITCOIN_CONTROLLER_CONF" | grep "miner_address =" | cut -d ' ' -f 3 | sed -r 's/"//g')"
+
+   log "  Importing miner address $MINER_ADDR..."
+   
+   echo "Try importing miner address $MINER_ADDR" >>"$FAUCET_LOGFILE"
+   bitcoin-cli -conf="$BITCOIN_CONF" importaddress "$MINER_ADDR" >>"$FAUCET_LOGFILE" 2>&1
+   RC=$?
+   if [ $RC -ne 0 ]; then 
+      logln "FAILED!"
+      return 1
+   fi
+
+   logln "ok"
+
+   log "Starting bitcoin faucet..."
+   faucet.sh "$FAUCET_PORT" "$BITCOIN_CONF" >"$FAUCET_LOGFILE" 2>&1 &
+   local FAUCET_PID=$!
+   logln "PID $FAUCET_PID"
+
+   echo "$FAUCET_PID"
+   return 0
+}
+
+start_stacks_master_node() {
+   logln "Setting up Stacks master node..."
+   test -f "$STACKS_MASTER_CONF".in || exit_error "No such file or directory: $STACKS_MASTER_CONF.in"
+
+   log "  Generating Stacks master config file..."
+   sed -r \
+      -e "s!@@STACKS_CHAINSTATE_DIR@@!$STACKS_CHAINSTATE_DIR!g" \
+      -e "s!@@STACKS_PUBLIC_IP@@!$STACKS_MASTER_PUBLIC_IP!g" \
+      "$STACKS_MASTER_CONF".in \
+      > "$STACKS_MASTER_CONF"
+   logln "ok"
+
+   log "Starting Stacks master node..."
+   BLOCKSTACK_DEBUG=1 RUST_BACKTRACE=full stacks-node start --config="$STACKS_MASTER_CONF" >"$STACKS_MASTER_LOGFILE" 2>&1 &
+   local STACKS_PID=$!
+   logln "PID $STACKS_PID"
+
+   echo "$STACKS_PID"
+   return 0
+}
+
+start_stacks_miner_node() {
+   logln "Setting up Stacks miner node..."
+   test -f "$STACKS_MINER_CONF".in || exit_error "No such file or directory: $STACKS_MINER_CONF.in"
+
+   log "  Generating Stacks miner config file..."
+   local STACKS_MASTER_IP="$1"
+   local BITCOIN_IP="$2"
+   local FAUCET_URL="$3"
+   local STACKS_MINER_SEED="$(blockstack-cli generate-sk | jq -r '.secretKey')"
+
+   sed -r \
+      -e "s/@@STACKS_MASTER_IP@@/$STACKS_MASTER_IP/g" \
+      -e "s/@@BITCOIN_IP@@/$BITCOIN_IP/g" \
+      -e "s/@@STACKS_MINER_SEED@@/$STACKS_MINER_SEED/g" \
+      -e "s!@@STACKS_CHAINSTATE_DIR@@!$STACKS_CHAINSTATE_DIR!g" \
+      "$STACKS_MINER_CONF".in \
+      > "$STACKS_MINER_CONF"
+   logln "ok"
+
+   local BTCADDR="$(blockstack-cli --testnet addresses "$STACKS_MINER_SEED" | jq -r '.BTC')"
```

Comment:
Is `--testnet` overriden by the `regtest` in the config? if so, should we just remove this flag?